docs/language/values-and-types.mdx

```diff
@@ -232,6 +232,49 @@ Fixed-Point numbers have multiple built-in functions you can use.
   fix.toBigEndianBytes()  // is `[0, 0, 0, 0, 7, 84, 212, 192]`
   ```
 
+## Minimum and maximum values
+
+The minimum and maximum values for integer and fixed-point number types are available through the fields `min` and `max`.
```

Comment:
Is it worth mentioning they are available for **all** integer and fixed point numbers? Or giving more examples?

Reply:
Clarified in 5de30cdee2e4918197fadab2e2356a7f66e71713